cloud-chatops/README.md

```diff
@@ -1,27 +1,116 @@
-This application is designed to help promote the closing of GitHub pull requests either by getting them approved and merged or closed when they go stale.<br>
-In principle, the app will notify authors about their pull requests until they are closed.<br>
+# Cloud ChatOps
+![Build](https://github.com/stfc/cloud-docker-images/actions/workflows/cloud_chatops.yaml/badge.svg)
+[![codecov](https://codecov.io/gh/stfc/cloud-docker-images/graph/badge.svg?token=BZEBAE0TQD)](https://codecov.io/gh/stfc/cloud-docker-images)
+
+## Contents:
+[About](#about)<br>
+[Usage / Features](#usage--features)<br>
+[Deployment](#deployment)<br>
+[requirements](#requirements)<br>
+
+### About
+
+Cloud ChatOps is designed to help promote the closing of GitHub pull requests. Either by getting them approved and merged or closed when they go stale.<br>
+The app will notify authors about their pull requests until they are closed / merged. There are multiple methods of sending these reminders.<br>
+
+### Usage / Features
+
+The main purpose of this application is to remind the team about pull requests they have open across their GitHub repositories.<br>
+This is facilitated by the below features which can be found in [main.py](src/main.py).
+
+#### Slash Commands:
+These slash commands can be run in any channel the application has access to.<br>
+ - `/prs <mine | all>`: Sends a private message to the user with a list of open pull requests. Either user authored or by anyone.
+
+#### Scheduled Events:
+Using the [schedule](https://pypi.org/project/schedule/) library functions are triggered on a weekly basis.<br>
+Events are defined in the `schedule_jobs` function:<br>
+- `run_global_reminder()`: Sends a message to the pull request channel with every open pull request across the repositories in a thread.
+- `run_personal_reminder()`: Sends a message to each user directly with a thread of their open pull requests.
+
 ### Deployment
-The entry point for is in [main.py](src/main.py) which will run the application.<br>
-The required files (below) need to be in the [cloud_chatops]() directory.<br>
-### Functionality
-As of current, the application gets all open pull requests from any Cloud owned repository and will send a message to our pull-request channel about each pull request notifying the author.<br>
-The app runs on an asynchronous loop scheduling each reminder to be sent out on days of our catch-ups (Monday, Wednesday and Friday).<br>
-On Mondays the application will mention users with **@** as to notify them directly. However, on the other 2 days authors will not be mentioned as to not spam people.<br>
+
+Secrets should always be stored in `$HOME/cloud_chatops_secrets` on the host system. Find out more about secrets [here](#Secrets)<br>
+The application can be run from a Docker image or source code. (Assuming running from project root)<br>
+
+#### Docker image:
+You can build the image locally or pull from [STFC Harbor](https://harbor.stfc.ac.uk).<br>
```

Comment:
We should link directly to the repo instead of Harbor generically then leave someone to dig around